picard/tagger.py

```diff
@@ -146,7 +146,7 @@ def __init__(self, picard_args, unparsed_args, localedir, autoupdate):
         )
 
         self._cmdline_files = picard_args.FILE
-        self._autoupdate = autoupdate
+        self.autoupdate_enabled = True
```

Comment:
Is this `True` setting left over from your testing?  Should it be `self.autoupdate_enabled = autoupdate` instead?

Reply:
Yeah, it was only for testing. Thank you.